modules/replicator/src/main/java/org/apache/ignite/internal/replicator/Replica.java

```diff
@@ -73,4 +74,11 @@ public interface Replica {
      * Shutdowns the replica.
      */
     CompletableFuture<Void> shutdown();
+
+    /**
+     * Updates peers and learners lists in raft client.
+     *
+     * @param configuration Peers and learners configuration.
+     */
+    void updateConfiguration(PeersAndLearners configuration);
```

Comment:
Could you rename it to something like "updateReplicationClientConfiguration"? Because in the scope of Replica the meaning of "configuration" is not obvious.

Reply:
Due to an onlibe discussion it is renamed to `updatePeersAndLearners`